src/net/rpc.rs

```diff
@@ -2087,14 +2089,22 @@ impl ConversationHttp {
         chainstate: &StacksChainState,
         query: MemPoolSyncData,
         max_txs: u64,
+        page_id: Option<Txid>,
     ) -> Result<Option<StreamCursor>, net_error> {
         let response_metadata = HttpResponseMetadata::from(req);
         let response = HttpResponseType::MemPoolTxStream(response_metadata);
         let height = chainstate
             .get_stacks_chain_tip(sortdb)?
             .map(|blk| blk.height)
             .unwrap_or(0);
-        let stream = Some(StreamCursor::new_tx_stream(query, max_txs, height));
+
+        test_debug!(
+            "Begin mempool stream from height {} page_id {:?} for up to {} txs",
+            height,
+            &page_id,
+            max_txs
+        );
```

Comment:
This also seems useful as a `debug!` (and k-v formatted)

Reply:
Fixed

Reply:
I think you need to push?